llvm/lib/Transforms/Coroutines/CoroSplit.cpp

```diff
@@ -2013,6 +2104,10 @@ splitCoroutine(Function &F, SmallVectorImpl<Function *> &Clones,
   buildCoroutineFrame(F, Shape, MaterializableCallback);
   replaceFrameSizeAndAlignment(Shape);
 
+  IRBuilder<> Builder(M.getContext());
```

Comment:
```suggestion
  IRBuilder<> Builder(F.getContext());
```

then we can get rid of `Module`.